application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue

```diff
@@ -185,10 +185,11 @@ export default {
         dropdownParent: 'body',
         hideSelected: true,
         renderMenuItem: function (item) {
-          this.getUserInformation(item.name);
-          const avatar = this.userAvatar;
+          chatServices.getUserInfo(item.name).then((data) => {
+            this.userAvatar = data.avatar ;
+          });
           const defaultAvatar = '/chat/img/room-default.jpg';
-          return `<img src="${avatar}" onerror="this.src='${defaultAvatar}'" width="20px" height="20px">
+          return `<img src="${this.userAvatar}" onerror="this.src='${defaultAvatar}'" width="20px" height="20px">
                       ${chatServices.escapeHtml(item.fullname)}<span style="float: right" class="chat-status-task chat-status-'+item.status+'"></span>`;
         },
         /* eslint-disable no-template-curly-in-string */
```

Comment:
Is this change needed ?

Reply:
> Is this change needed ?

Yes, is needed to have the value of the usersAvatar without having an exception with the promised result. discussed with @AzmiTouil